### ISSUE TITLE
Running vagrant up results in Stderr: VBoxManage: error: RawFile#0 failed to create the raw output file /Users/ehankinson/Projects/leandog/atdd-linux-vagrant-master/ubuntu-xenial-16.04-cloudimg-console.log (VERR_FILE_NOT_FOUND)

### ISSUE THREAD

**Issue by danielhelfand** [leandog/atdd-linux-vagrant-vm#2]
When running `vagrant up` in the base directory of this repository on macOS Mojave, the following error appears:

```
There was an error while executing `VBoxManage`, a CLI used by Vagrant
for controlling VirtualBox. The command and stderr is shown below.

Command: ["startvm", "f0ecf372-c452-40dc-b072-d1fb6e3b9b1d", "--type", "gui"]

Stderr: VBoxManage: error: RawFile#0 failed to create the raw output file /Users/ehankinson/Projects/leandog/atdd-linux-vagrant-master/ubuntu-xenial-16.04-cloudimg-console.log (VERR_FILE_NOT_FOUND)
VBoxManage: error: Details: code NS_ERROR_FAILURE (0x80004005), component ConsoleWrap, interface IConsole
```

In working some Lean Dog team members, it appears the solution is to open Virtual Box (use `cmd + space` and type Virtual Box and open the Virtual Box application). Click on LDATDDDev, click on the Ports tab, and change the path from `/Users/ehankinson/Projects/leandog/atdd-linux-vagrant-master/ubuntu-xenial-16.04-cloudimg-console.log` to `/Users/<YOUR USER FOLDER>/<LOCAL PATH TO ATDD REPO>/atdd-linux-vagrant-vm/ubuntu-xenial-16.04-cloudimg-console.log`

**Comment by qre0ct** [comment]
There is a small hack that seemed to work in my case. Had the exact same issue. All I did was created the missing path (like so in your case) : /Users/ehankinson/Projects/leandog/atdd-linux-vagrant-master/ubuntu-xenial-16.04-cloudimg-console.log
It solves the problem. Still trying to figure out though how to change the logs path for VBoxManage.

Actually, just figured out. This can be fixed by changing the the `Path/Address` settings in the `ports` section of the virtual box's settings. Make it to the missing path (/Users/ehankinson/Projects/leandog/atdd-linux-vagrant-master/ubuntu-xenial-16.04-cloudimg-console.log in your case). Ensure that the box was shut down so you can make the change. 

<img width="622" alt="Screen Shot 2019-06-23 at 1 09 57 PM" src="https://user-images.githubusercontent.com/11993558/59973136-96eac480-95b8-11e9-99c9-96874b420cd4.png">

**Comment by kumichou** [comment]
The line in the Vagrantfile

```
vb.customize ["modifyvm", :id, "--uartmode1", "disconnected"]
```

is supposed to disconnect the UART and not require that location to be there. Apparently this isn't working in all versions of VirtualBox?

**Comment by lukmanlab** [comment]
May be in macos, need allow permission to document. You can test run another Vagrantfile and re-run your existing vagrant/restore snapshot existing vm.

**Comment by samfullman** [comment]
Yeah, that .customize method is not working in vagrant --version 2.2.2

**Comment by samfullman** [comment]
When I packaged my box, I had a path to that Serial Port that was relative to my computer, hence the message.  So the answer was, make sure that setting is corrected (as shown above) before doing a `vagrant package`

https://superuser.com/questions/1622167/vagrant-box-fails-trying-to-log-to-a-path-the-box-was-created-from/1622179#1622179

**Comment by Mosu6997** [comment]
> 

What is an ATDD Repository?
